applications/tune_psf.py

```diff
@@ -169,6 +169,14 @@ def loadData(datafile):
         modelVersion=args.model_version,
         label=label,
     )
+    # If we want to start from different values then the one in the model:
```

Comment:
Seems to be an incomplete sentence, what exactly do you mean here?

Reply:
I guess a better sentence would be `If we want to start from values different than the ones currently in the model`. Would that be clear (I don't explain further on purpose)?

Either way the test failed for so I will fix it first.

Reply:
Good.